src/main/java/eu/europeana/metis/sandbox/domain/RecordInfo.java

```diff
@@ -0,0 +1,32 @@
+package eu.europeana.metis.sandbox.domain;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Collections;
+import java.util.List;
+
+public class RecordInfo {
```

Comment:
This now is the return for each workflow step service, so that we can encapsulate non stoping errors that occured during processing